src/util/cliffThoughtHeight.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Calculates the height of a drop target at the cliff, decreasing with depth depth. This makes it easier to drop a thought at the desired level. Returns height in ems.
+ */
+const calculateCliffThoughtsHeight = ({ deepestDepth, depth }: { deepestDepth?: number; depth?: number }) => {
```

Comment:
Let's call this `calculateCliffDropTargetHeight`. It's a little more verbose, but it's more accurate.

We are now using `cliff` for the `deepestDepth`, so let's rename the parameter name to match. It's not good to use multiple identifier names for the same concept.